modules/installation-requirements-user-infra.adoc

```diff
@@ -140,7 +147,9 @@ On your z/VM instance, set up:
 [discrete]
 === Disk storage for the z/VM guest virtual machines
 
-* FICON attached disk storage (DASDs). These can be z/VM minidisks, fullpack minidisks, or dedicated DASDs. If available, use HyperPAV to ensure optimal performance.
+* FICON attached disk storage (DASDs). These can be z/VM minidisks, fullpack minidisks, or dedicated DASDs.
+ifdef::ibm-z[To reach the minimum required DASD size for RHCOS installations, you need extended address volumes (EAV).]
```

Comment:
Is this ifdef::ibm-z really necessary? It looks like it's already inside a big ibm-z ifdef on line 121.

Reply:
Good catch, removed the inner conditional.